tardis/plasma/properties/ion_population.py

```diff
@@ -163,8 +163,24 @@ def calculate(
 
     @staticmethod
     def get_zeta_values(zeta_data, ion_index, t_rad):
+        """
```

Comment:
The docstring should adhere to the tardis documentation standards.  We use PEP 8 as the standard for our docstrings.

Reply:
It should be more clear what the data types of the arguments are since this will help resolve the below comment.

Reply:
Have a look at https://tardis-sn.github.io/tardis/development/code_quality.html#docstrings for the docstrings. You can also use https://github.com/psf/black to automatically format your code.